www/include/eventLogs/xml/data.php

```diff
@@ -360,13 +365,15 @@
 
 $whereOutput = "";
 if (isset($output) && $output != "") {
-    $whereOutput = " AND logs.output like '%" . $pearDBO->escape($output) . "%' ";
+    $whereOutput = " AND logs.output like '%" . CentreonUtils::escapeSecure($output) . "%' ";
 }
 
 $innerJoinEngineLog = "";
 if ($engine == "true" && isset($openid) && $openid != "") {
-    $innerJoinEngineLog = " INNER JOIN instances i ON i.name = logs.instance_name AND i.instance_id IN (" .
-        $pearDBO->escape($openid) . ") ";
+    if (preg_match('/([0-9]+)?(,[0-9]+)*/', $openid, $matches)) {
```

Comment:
be carefull as instance_id == -1 means "all pollers" filter

Reply:
Done